apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -587,6 +588,8 @@ private boolean queryBeneExistsByPartDContractCodeAndYearMonth(
           "bene_exists_by_year_month_part_d_contract_id",
           beneHistoryMatchesTimerQueryNanoSeconds,
           matchingBeneExists ? 1 : 0);
+      // Close the resource
```

Comment:
not sure why you didn't follow the try w/resources paradigm; obviously SQ will flag this one that has no close, but the try with reources will automatically close the resource when it goes out of scope.

Reply:
For this particular use-case I went with closing the resource which is on Line 592. As we already had the finally block and was trying to not to have an additional try block in the code.